src/task2.js

```diff
@@ -0,0 +1,31 @@
+import csv from 'csvtojson';
+import { createReadStream, createWriteStream } from 'fs';
+import { pipeline } from 'stream';
+
+const inputFilePath='./csv/nodejs-hw1-ex1.csv';
+const outputFilePath='./csv/nodejs-hw1-ex1.txt';
+
+
+const readable = createReadStream(inputFilePath);
+const writable = createWriteStream(outputFilePath, 'utf8');
+
+// Solution 1
+// readable
+//   .pipe(csv())
+//   .on('error', err => console.log(err))
+//   .pipe(writable)
+//   .on('error', err => console.log(err));
+
+// Solution 2
+pipeline(
+  createReadStream(inputFilePath),
+  csv(),
```

Comment:
format of output file is wrong (csvtoJson wrongly parse because of wrong csv file format). Also it should have additional formatting(not listed in requirement, but visible in example file): 
1) `Amout` field should not persist
2) all fields should be in lower case
3) numbers should not be like strings

Take a look to `csvtojson` options object. It can cover all this formatting.